src/skmultiflow/utils/utils.py

```diff
@@ -179,3 +179,38 @@ def is_scalar_nan(x):
     # convert from numpy.bool_ to python bool to ensure that testing
     # is_scalar_nan(x) is True does not fail.
     return bool(isinstance(x, numbers.Real) and np.isnan(x))
+
+
+def add_dict_values(dict_a: dict, dict_b: dict, inplace=False) -> dict:
+    """ Adds two dictionaries, summing the values of elements with the same key.
+
+    This function iterates over the keys of dict_b and adds their corresponding
+    values to the elements in dict_a. If dict_b has a (key, value) pair that
+    does not belong to dict_a, this pair is added to the latter dictionary.
+
+    Parameters
+    ----------
+    dict_a: dictionary to update.
+    dict_b: dictionary containing the new observations.
```

Comment:
Suggestion: 
```
    dict_b: dictionary whose values will be added to `dict_a`.
```

Reply:
Addressed in [117dc8b](https://github.com/scikit-multiflow/scikit-multiflow/pull/183/commits/117dc8bddb3faad066df992197abe7ff0f876397).